pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
My view is that we must define a generic storage framework first. I don't want us adding technical debt by integrating custom storage drivers into the core code in this way. This is why I put up the design proposal for a generic system to integrate storage. The generic system does not have to be very much extra work to achieve, so I see no compelling reason to do this docker specific solution first.